src/test/java/com/mentoring/pageobject/MainPage.java

```diff
@@ -64,4 +73,20 @@ private void enterRegistrationEmailAddress(String emailAddress) {
     @Override
     public void waitUntilPageLoads() {
     }
+
+    public boolean invalidUserName() {
+        return isElementDisplayed(invalidUsername);
+    }
+
+    public boolean invalidPassword() {
+        return isElementDisplayed(invalidPassword);
+    }
+
+    public boolean invalidEmailAddress() {
+        return isElementDisplayed(invalidEmail);
+    }
+
+    public boolean isSubmitButtonEnabled() {
+        return submitRegistrationButton.isEnabled();
```

Comment:
Please write a custom method for checking element state (name somethig like: isElementEnabled). This gives us possibility for customization of basic operations like this (e.g: add logging, exception handling, method overloading etc.)